Kernel/Memory/MemoryManager.cpp

```diff
@@ -416,7 +416,7 @@ UNMAP_AFTER_INIT void MemoryManager::initialize_physical_pages()
 {
     m_global_data.with([&](auto& global_data) {
         // We assume that the physical page range is contiguous and doesn't contain huge gaps!
-        PhysicalAddress highest_physical_address;
+        PhysicalAddress highest_physical_address = PhysicalAddress { 0x100000000LL };
```

Comment:
I'd like to see a comment that says that some devices like the LAPIC maybe found at almost 4GiB so this is a thing that might be x86-specific as well.